core/src/test/scala/org/apache/spark/DistributedSuite.scala

```diff
@@ -323,7 +323,7 @@ class DistributedSuite extends SparkFunSuite with Matchers with LocalSparkContex
     data.persist(StorageLevel.MEMORY_ONLY_2)
     data.count
     assert(sc.persistentRdds.isEmpty === false)
-    data.unpersist()
+    data.unpersist(blocking = true)
```

Comment:
This is an example of tests where blocking behavior seems important, so they now explicit request it.